### ISSUE TITLE
Details about the observation experiments

### ISSUE THREAD

**Issue by slyviacassell** [tmllab/PES#1]
Dear authors,
    You did a simple and good work on label noise. I have read this publication and did not find the training details about the observation experiment in the introduction. Would you mind providing the training details, such as `the epoch for training the reset layers` after re-initialization, `learning rate`, `optimizer`, etc? Thanks a lot!

**Comment by bybeye** [comment]
Thank you for your interest in our work. The experimental settings for the evaluation in Figure 1 are consistent with those outlined in Section 3.1, except for the number of training epochs.

Here are the specific parameters:
Training epochs: 100 epochs
Batch size: 128
Optimizer: SGD
Learning rate: 0.1
Weight decay: 1e-4
Momentum: 0.9

**Comment by slyviacassell** [comment]
Thanks for your reply.
